IHP/Test/Database.hs

```diff
@@ -34,8 +38,11 @@ createTestDatabase databaseUrl = do
             |> cs
 
     libDir <- LibDir.findLibDirectory
-    importSql newUrl (cs libDir <> "/IHPSchema.sql")
-    importSql newUrl "Application/Schema.sql"
+
+    let importSql file = Process.callCommand ("psql -h '" <> currentDir <> "/build/db' -d " <> (cs databaseName) <> " < " <> file)
```

Comment:
I think this should be using `databaseUrl` instead of the current dir, otherwise the function might fail if the database url passed to this function is different than the current directory

```suggestion
    let importSql file = Process.callCommand ("psql -h '" <> databaseUrl <> "/build/db' -d " <> (cs databaseName) <> " < " <> file)
```

Reply:
Agree using `newUrl` which will have the temporary database name patched over the connection from databaseurl